### PR TITLE
[WIP] Add a simple visualization using real experiment data (closes #3209)

### LINKED ISSUE #3209: Create a simple analysis visualization in the Experiment summary page
This will be compiled in Svelte (so that the graph paper capabilities can be used) and integrated into the experiment summary page.

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/EXP-229)

### PR DESCRIPTION
**Pull request by emtwo**
WIP

Will need the following items:
- [x] Add visualization package compiled in Svelte.
- [x] Change name from pibal to visualization
- [x] Read data from remote gcs bucket
- [x] Pipe in CI integration with linting
- [x] Wrap in a feature flag
- [x] Polish/simplify rollup config and package.json
- [x] Use typescript in svelte
- [x] Create 1 graph with real data

### CODE REVIEW

**Comment by jaredlockhart** [comment]
🎉 🎉 🎉

**Comment by emtwo** [comment]
for #3209

File: app/experimenter/static/visualization/package.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    "tsconfig-paths": "^3.9.0",
+    "tslib": "^2.0.1",
+    "typescript": "^4.0.2"
+  },
+  "dependencies": {
+    "@google-cloud/bigquery": "^4.7.0",
+    "express": "^4.17.1",
+    "sirv-cli": "^0.4.4"
+  }
+}
```
Comment (pdehaan) [inline app/experimenter/static/visualization/package.json:66]: We might need to tweak the .github/dependabot.yml file to make sure dependabot depends these deps up to date.

**Comment by emtwo** [comment]
this PR is no longer needed

**Pull request closed without merging**